docs/manual/spring-security-docs-manual.gradle

```diff
@@ -30,6 +33,15 @@ docsZip {
 	}
 }
 
+htmlSanityCheck {
```

Comment:
Can this be removed in favor of what you've added in `DocsPlugin`?

Reply:
If we define all the properties in the `DocsPlugin` will it remain reusable in other projects? If we have different folders, files, etc, from project to project, for instance

Reply:
> dependsOn asciidoctor

Since `DocsPlugin` requires Asciidoctor, it somewhat implied that its other applications depend on `asciidoctor`. Note that this is what `docsZip` does in `DocsPlugin`.

> sourceDir = new File("$buildDir/docs/asciidoc")

Additionally, I'd imagine that if someone needs to change where the `asciidoctor` task outputs its files, `DocsPlugin` would be able to look that up and apply it to the `HtmlSanityCheck` configuration.

> include "index.html"

This might be project-specific, but I wonder if you can include all files in the source directory instead?

Reply:
Yeah, it makes sense. If another module wants to use a different set of properties, it's just a matter of overriding the default from the `DocsPlugin`